targets/TARGET_Cypress/TARGET_PSOC6/TARGET_CY8CPROTO_064_SB/device/COMPONENT_CM4/TFM_partition/flash_layout.h

```diff
@@ -0,0 +1,225 @@
+/*
```

Comment:
Since `flash_layout.h` and `region_defs.h` are common to both secure and non-secure components, can we move these files under `TARGET_CY8CPROTO_064_SB`? Also, can we change `TFM_partition` to `partition` to keep this in sync with other PSA targets?

Reply:
Fixed